docs/UserGuide.md

```diff
@@ -465,13 +465,87 @@ StaffConnect data are saved in the hard disk automatically after any command tha
 StaffConnect data are saved automatically as a JSON file `[JAR file location]/data/staffconnect.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, StaffConnect will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
+If your changes to the data file makes its format invalid, StaffConnect will discard all data and start with an empty data file at the next run. Hence, it is **recommended to take a backup** of the file before editing it.<br>
 Furthermore, certain edits can cause StaffConnect to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). 
-The application will not prompt the user if the format of the data file is incorrect, but instead provide the user with an empty staff book.
-Therefore, edit the data file only if you are confident that you can update it correctly.
+The application will not prompt the user if the format of the data file is incorrect, but instead provide the user with an empty staff book.<br>
+**Therefore, edit the data file only if you are confident that you can update it correctly.**
 
 </div>
 
+#### Valid Data Values in `persons`
+
+All attributes' restrictions except **Favourite** can be found in the [Attribute Summary](#attribute-summary) table.
+
+Attribute | Data Field | Valid Data Values
+----------|--------|-------------
+Name | `name` | `Alice`, `Ben10`
+Phone | `phone` | `123`, `98765432`
+Email | `email` | `e@123.com`, `hello@h-h.com`, `one+two@h-h.hh`, `hello@e-h.e-hh`
+Module | `module` | `gess1025`, `hsi1000`, `CS2103T`
+Faculty | `faculty` | Refer to the [Valid Faculty Values](#valid-faculty-values) table.
+Venue | `venue` | `COM4-02-33`, `LT21`, `Kent Ridge Vale, Tulip Street, #12-34`
+Tag | `tags` | `professor`, `Number1TA`
+Availabilitity | `availabilities` | `TUESDAY 12:00 13:00`, `WEDNESDAY 12:00 13:00`
+Meeting | `meetings` | Refer to the [Valid Data Values in meetings](#valid-data-values-in-meetings) table.
+Favourite | `favourite` | `Favourite` or `Not favourite`
+
+#### Valid Data Values in `meetings`
```

Comment:
suggestion: would it be better for these subheadings to specify that `Valid Data Values in persons` -> `Valid Data Values for Person in JSON file`?
same for meetings subheader

Reply:
I think that's a good suggestion. It will be more informative to the users at the beginning as it is in the table of content.